src/Compilers/Core/Portable/CommandLine/AnalyzerConfigSet.cs

```diff
@@ -190,6 +190,18 @@ public AnalyzerConfigOptionsResult GetOptionsForSourcePath(string sourcePath)
             // we would have merged them earlier.
             foreach (var section in _globalConfig.NamedSections)
             {
+                if (!PathUtilities.IsUnixLikePlatform
+                    && section.Name.Length > 2
+                    && normalizedPath.Length > 2
+                    && PathUtilities.IsDirectorySeparator(normalizedPath[1])
+                    && PathUtilities.IsDirectorySeparator(section.Name[1])
```

Comment:
Wouldn't we want the second character to be a colon `:` instead? And the third letter to be a directory separator?

Reply:
Yup, fixed!